modular_skyrat/master_files/code/modules/mod/mod_types.dm

```diff
@@ -1,4 +1,5 @@
 /obj/item/mod/control/pre_equipped/nuclear/chameleon
+	req_access = null
```

Comment:
Wouldn't it make more sense to instead give lone infiltrators syndicate access?

Reply:
Making it consistent with traitor modsuits